libs/ui-toolkit/src/utils/intent.ts

```diff
@@ -20,9 +21,9 @@ export const getIntentBackground = (intent?: Intent) => {
   return {
     'bg-neutral-200 dark:bg-neutral-800': intent === undefined,
     'bg-black dark:bg-white': intent === Intent.None,
-    'bg-vega-pink dark:bg-vega-yellow': intent === Intent.Primary,
-    'bg-danger': intent === Intent.Danger,
-    'bg-warning': intent === Intent.Warning,
+    'bg-vega-blue-300 dark:bg-vega-blue-700': intent === Intent.Primary,
+    'bg-vega-pink': intent === Intent.Danger,
```

Comment:
Anyone with any ideas why this change made a difference? I couldn't get a colour back from `bg-danger` even though I can see in the theme file it should be defined as the pink. Changing this line to `bg-vega-pink` made passing `Intent.Danger` to this function apply a colour properly. Pretty confused!

Reply:
I just tried making the same changes as you and couldn't recreate the problem, this has happened to me and it was because I had added the classname in a file which wasn't included in the tailwind.config.js. Could that be the case here?

Reply:
I suppose that's possible, but I'm then confused as to why e.g. `bg-vega-pink` works in the same place! 🤔

Reply:
I dropped this change out of the PR